core/src/main/java/org/apache/iceberg/actions/RewriteFileGroup.java

```diff
@@ -59,6 +62,12 @@ public Set<DataFile> rewrittenFiles() {
         .collect(Collectors.toCollection(DataFileSet::create));
   }
 
+  public Set<DeleteFile> orphanedDVs() {
```

Comment:
I think we call this dangling deletes previously.

Reply:
I'm OK either way with a slight preference for `orphaned`; it is inconsistent with the previous naming convention of dangling but I do think it's a much clearer name indicating the state of the DV.

Reply:
Can these DVs be physically removed like orphaned files now? I think they are just removed from the current snapshot.